src/main/java/org/aksw/simba/squirrel/analyzer/impl/Guess.java

```diff
@@ -0,0 +1,38 @@
+package org.aksw.simba.squirrel.analyzer.impl;
+
+public class Guess {
+
+	public static void parse(String fileContent)

```

Comment:
I won't comment this method since I assume that you want to use the other class. Please remove this class if it is not being used, anymore.